packages/edit-post/src/components/back-button/style.scss

```diff
@@ -31,7 +32,7 @@
 			position: absolute;
 			top: 9px;
 			right: 9px;
-			bottom: 9px + $border-width; // Height of toolbar in edit-post (not edit-site) is 61px tall.
+			bottom: 9px;
```

Comment:
This is not directly related to the original issue, but just a minor tweak to make the focus outline square.

|Before|After|
|-|-|
| <img width="350" height="160" alt="Image" src="https://github.com/user-attachments/assets/3c4d6c38-bf33-41ba-922e-0ab8ab966c61" /> | <img width="350" height="160" alt="Image" src="https://github.com/user-attachments/assets/51ce2ec7-695a-4d2b-914d-9e8a01311160" /> |